css/style.css

```diff
@@ -106,3 +105,313 @@ h4 {
   color: #344563;
   margin: 12px 24px 12px 24px;
 }
+
+.my-works {
+  margin: 14px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  justify-content: baseline;
+}
```

Comment:
Please, I recommend that you stick to the margins between the projects given in the design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/192076066-e5c36bcb-0dfe-4f8f-b900-c1b826f43235.png)

Figma Design:
![image](https://user-images.githubusercontent.com/20554583/192076077-0347b221-dbde-41d0-93c2-6fed90acaf4e.png)